apiserver/src/error.rs

```diff
@@ -16,6 +16,9 @@ pub enum Error {
     #[snafu(display("Unable to parse HTTP header. Missing '{}'", missing_header))]
     HTTPHeaderParse { missing_header: &'static str },
 
+    #[snafu(display("Unable to detect cluster IP family. For '{}'", source))]
```

Comment:
The message isn't "generic" enough for the name of this error, I think it should be something along the lines of `Missing env var <>`. Or else, have an error that is specific for this message `MissingClusterIPFamiliy`.